release/preview/ubuntu16.04/docker/Dockerfile

```diff
@@ -52,7 +52,8 @@ RUN apt-get update \
     && apt-get install -y --no-install-recommends \
     powershell-preview=${PS_VERSION}${PS_VERSION_POSTFIX} \
     && apt-get clean \
-    && rm -rf /var/lib/apt/lists/*
+    && rm -rf /var/lib/apt/lists/* \
+    && ln -sf /opt/microsoft/powershell/6-preview/pwsh /usr/bin/pwsh
```

Comment:
Here and in other dockerfiles:
why are we creating 'pwsh' symlink that is pointing to the 'preview' version of PS?

Reply:
We know that no stable version is in this container.  CloudShell did something similar.  I thought it was a good idea.